### ISSUE TITLE
Codes vs controlled vocabularies

### ISSUE THREAD

**Issue by ahwagner** [ave-dcd/mave_vocabulary#3]
We currently provide a controlled vocabulary and should also include mappings from that vocabulary to concept codes where appropriate. We describe the use of multiple ontologies that are to be used:
https://github.com/ave-dcd/mave_vocabulary/blob/96d3dcc0f59330ddd1d2ddcbf0b5ef9ff54a5cc4/README.md?plain=1#L101-L112

But our schema only provides the ability for codings to be used for `phenotypicAssay.relevance` (required; we recommend MONDO) and the `phenotypicAssay.modelSystem.identifiers` (not required; we recommend the cell line ontology). We should update this section to either state that we provide a controlled vocabulary that _maps to_ an ontology (and provide a SSSOM mapping; easy), or we update our schema to _require_ the use of `coding` objects (as we do `phenotypicAssay.relevance`).

**Comment by afrubin** [comment]
I agree with this and think it merits further discussion. We should also revisit this in the context of switching over to [LinkML](https://linkml.io/), which was out of scope for our initial version of this work but might be a better fit long term. The LinkML representation seems well-suited to this use case, and is also gaining traction.
